CONTRIBUTING.md

```diff
@@ -21,6 +21,53 @@ For your contribution to be accepted you will need to sign the [Shopify Contribu
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create a new Pull Request
 
+## Run Language Server 
+
+If you're making changes to the language server and you want to debug, you can run the repo's version of theme-check-language-server
+
+### VS Code
+
+* Install the [Shopify Liquid](https://github.com/shopify/theme-check-vscode) plugin
+* At minimum add the following to `settings.json` (be sure to update `shopifyLiquid.languageServerPath` with your local path to theme-check)
+
+ ```json
+"shopifyLiquid.formatterDevPreview": true,
+"shopifyLiquid.languageServerPath": "/path/to/theme-check/bin/theme-check-language-server",
+ ```
+
+ * Setup your local environment by running the following script in your console from the root of theme-check
+
+ ```bash
+#!/usr/bin/env bash
+cd "$HOME/src/github.com/Shopify/theme-check" &> /dev/null
+
+chruby () {
+  source '/opt/dev/sh/chruby/chruby.sh'
+  chruby "$@"
+}
+
+export THEME_CHECK_DEBUG=true
+export THEME_CHECK_DEBUG_LOG_FILE="/tmp/theme-check-debug.log"
+touch "$THEME_CHECK_DEBUG_LOG_FILE"
+chruby 2.7 &>/dev/null
+gem env &>/dev/null
```

Comment:
The way I have it setup is that this is actually a script file that I then use directly instead of having the languageServerPath set to the theme-check repo's bin file.

Something like:

1. Create an executable wrapper of theme-check-language-server in your `$HOME/bin` folder named `theme-check-language-server`.

```bash
# Make sure you have a $HOME/bin
mkdir -p $HOME/bin

# Paste the script in $HOME/bin/theme-check-language-server
cat <<-'EOF' > $HOME/bin/theme-check-language-server
#!/usr/bin/env bash
cd "$HOME/src/github.com/Shopify/theme-check" &> /dev/null
chruby () {
  source '/opt/dev/sh/chruby/chruby.sh'
  chruby "$@"
}
export THEME_CHECK_DEBUG=true
export THEME_CHECK_DEBUG_LOG_FILE="/tmp/theme-check-debug.log"
touch "$THEME_CHECK_DEBUG_LOG_FILE"
chruby 2.7 &>/dev/null
gem env &>/dev/null
bin/theme-check "$@"
EOF

# make it executable
chmod u+x $HOME/bin/theme-check-language-server
```

2. Use that in your `shopifyLiquid.languageServerPath` configuration (and replace `<YOUR_USERNAME>` by your username).

```
"shopifyLiquid.formatterDevPreview": true,
"shopifyLiquid.languageServerPath": "/Users/<YOUR_USERNAME>/bin/theme-check-language-server",
```